src/app/work-item/work-item-detail/work-item-detail.component.ts

```diff
@@ -324,7 +324,7 @@ export class WorkItemDetailComponent implements OnInit, AfterViewInit, OnDestroy
   getAllUsers(): Observable<any> {
     return Observable.combineLatest(
       this.userService.getUser(),
-      this.userService.getAllUsers()
+      this.workItemService.getCollaborators()
```

Comment:
@sbose78 That's  bit tricky to explain. I'll raise a PR on your fork / this branch.